src/braille/internal/braille.cpp

```diff
@@ -535,7 +535,7 @@ void BrailleEngravingItemList::addEngravingItem(EngravingItem* el, const QString
     //Manual doubling slashes '\' because Regex doesn't work. Don't know why.
     QString txt = QString();
     for (int i=0; i < braille.length(); i++) {
-        if (braille.at(i) == '\\') {
+        if (braille.at(i) == QString('\\')) {
```

Comment:
The declaration is:
`const QChar at(qsizetype position) const`
Hence this should have been
`if (braille.at(i) == QChar('\\')) {`
Or even, if it does not break the style,
`if (braille[i] == QChar('\\')) {`